kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ParquetFileWriter.java

```diff
@@ -131,12 +142,12 @@ private Optional<DataFileStatus> writeNextFile() {
 
                 Path filePath = generateNextFilePath();
                 assert batchWriteSupport != null : "batchWriteSupport is not initialized";
-                try (ParquetWriter<Integer> writer =
-                             createWriter(filePath, batchWriteSupport)) {
+                try (ParquetWriter<Integer> writer = createWriter(filePath, batchWriteSupport)) {
                     boolean maxFileSizeReached;
                     do {
                         consumeNextRow(writer);
-                        maxFileSizeReached = writer.getDataSize() >= targetMaxFileSize;
+                        maxFileSizeReached =
```

Comment:
nit but a clarifying comment here might be helpful

Reply:
added.